### PR TITLE
As per twigphp/Twig#472, automatic escaping is not yet available

### PR DESCRIPTION
**Pull request by Ocramius**
While it would be a cool feature, it is also impossible to apply automatic escaping in most cases.
Advertising this as a out-of-the-box feature is problematic, as it probably mislead multiple developers
that are using the tool without applying proper contextual escaping.

Ref: twigphp/Twig#472

### CODE REVIEW

**Comment by WouterJ** [comment]
Well, it is automatic in a way that it always escapes your variables unless you use the `|raw` filter.

**Comment by stof** [comment]
We should keep the mention of ``automatic output escaping`` though (contextual escaping is indeed not supported by the auto-escaper)

**Comment by Ocramius** [comment]
Then I got mislead by the wording. I'll reword it to fix my misconception.

**Comment by WouterJ** [comment]
Thanks Marco!
